.github/workflows/teardown-frontend-dev.yml

```
@@ -0,0 +1,332 @@
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Install Amplify CLI
+        run: |
+          npm install -g @aws-amplify/cli
+          amplify --version
+
+      - name: Destroy frontend infrastructure
+        id: destroy
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:45:36: Double quote to prevent globbing and word splitting

(shellcheck)